examples/checkPars.py

```diff
@@ -0,0 +1,162 @@
+import lsst.afw.image as afwImage
+import lsst.afw.math as afwMath
+import lsst.afw.table as afwTable
+import lsst.meas.algorithms as measAlg
+import lsst.meas.base as measBase
+
+"""
+checkPars.py is a python program which can be called as a command line script.
+It is used to print the distribution of a library over the entire
+LSST focal plane, divided into groups (currently by raft)
+"""
+
+#   This routine takes a list of images and a catalog name (presumably covering the
+#   same area as the images.  Any entry in the catalog whose center is contained in
+#   the bounding rectangle of a given image is counted as belonging to the h
+#   raft and the sensor of the image.
+def getRaftStatistics(images, catname, raft):
+    imageinfo = []
+    #   First create a list of "imageinfo" entries which know the position
+    #   and bounding box of each raft, sensor.
+    for i, image in enumerate(images):
+        exp = afwImage.ExposureF(image)
+        width, height = exp.getMaskedImage().getImage().getArray().shape
+        llcoord = exp.getWcs().pixelToSky(afwGeom.Point2D(0,0))
+        urcoord = exp.getWcs().pixelToSky(afwGeom.Point2D(width-1, height-1))
+        ramin = llcoord.toFk5().getRa().asDegrees()
+        ramax = urcoord.toFk5().getRa().asDegrees()
+        if ramin > ramax:
+            temp = ramin
+            ramin = ramax
+            ramax = temp
+        decmin = llcoord.toFk5().getDec().asDegrees()
+        decmax = urcoord.toFk5().getDec().asDegrees()
+        if decmin > decmax:
+            temp = decmin
+            decmin = decmax
+            decmax = temp
+        # Get the raft and sensor values from the imagename.  
+        ind = image.find("_R")
+        raft = image[ind+1:ind+5]
+        ind = image.find("_S")
+        sensor = image[ind+1:ind+5]
+        imageinfo.append((raft, sensor, ramin, ramax, decmin, decmax))
+    #   hist is a counting array, one entry per image
+    hist = numpy.zeros(len(images), dtype = int)
+
+    # read through the OpSim catalog and count how many are in each raft, sensor
+    cat = open(catname, "r")
+    for line in cat:
+        # catalog lines with objects have x,y position at index 2,3.  Use that for 1st guess
+        attrs = line.split()
+        if attrs[0] != "object": continue
+        ra = float(attrs[2])
+        dec = float(attrs[3])
+        for i, info in enumerate(imageinfo):
+            ramin = info[2]
+            ramax = info[3]
+            decmin = info[4]
+            decmax = info[5]
+            if ra <= ramax and ra >= ramin and dec <= decmax and dec >= decmin:
+                hist[i] = hist[i] + 1
+    for i, info in enumerate(imageinfo):
+        print "    ", hist[i], imageinfo[i]
+    print hist[0], hist.sum()
+    return hist.sum()
+
+#   This main program does counts from images created by PhoSim, over rafts and sensors
+#   Each raft is comprised of 9 files named basename_Rnn_Smm.fits in the input directory
+if __name__ == "__main__":
+    """
```

Comment:
I don't think you need both this documentation and the help strings with the arguments.  I'd just keep the latter, otherwise both have to be updated.